### PR TITLE
[11.0] connector_jira_servicedesk: Add servicedesk_issue_url

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

File: connector_jira_servicedesk/models/account_analytic_line/common.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    jira_servicedesk_issue_url = fields.Char(
+        string='Original JIRA issue Link',
+        compute='_compute_jira_servicedesk_issue_url',
+    )
+
+    @api.depends('jira_issue_key')
+    def _compute_jira_servicedesk_issue_url(self):
+        """Compute the external URL to JIRA service desk."""
+        for record in self:
+            jira_project = self.project_id.jira_bind_ids[0]
```
Comment (jcoux) [inline connector_jira_servicedesk/models/account_analytic_line/common.py]: Are you sure to always have a project and a binding for it?
If not, your code will can raise.

File: connector_jira_servicedesk/models/account_analytic_line/common.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            record.jira_servicedesk_issue_url = jira_project.make_servicedesk_issue_url(
+                record.jira_issue_key
+            )
```
Comment (jcoux) [inline connector_jira_servicedesk/models/account_analytic_line/common.py]: Don't check that `jira_issue_key` is set?
Field is not defined as required.

**Review by jcoux — APPROVED** [review]
Technically seems good :+1:

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**